src/pages/home/sidebar/SidebarLinks.js

```diff
@@ -89,6 +89,9 @@ const SidebarLinks = (props) => {
         ? [styles.sidebarHeader, styles.sidebarHeaderActive]
         : [styles.sidebarHeader];
 
+    if (_.isEmpty(props.reports) || _.isEmpty(props.personalDetails)) {
```

Comment:
I'm not exactly clear on:
1. What `SidebarLinks`'s purpose is -- can we add a comment maybe on line 84 briefly saying what it's for? 
2. What this added `if` is here for -- perhaps a comment would help?

Reply:
+1 to `2` 

Why return `null` here? Will something break if we don't? I think it's usually better to allow views to present themselves with or without the data we need rather than return `null` - but maybe there's a reason?

Reply:
> Why return null here?
I added it just to reduce the number of renders. When we sign into the app, it renders this component 10 times without report or personalDetails values and hence doesn't really do anything.

> Will something break if we don't?
If we don't or do return null the component seems to work fine in both cases. I don't see any errors. Having null just reduced the number of times we updated the return. 

> I think it's usually better to allow views to present themselves with or without the data we need rather than return null - but maybe there's a reason?
Sure I think that's fine. I guess i thought reducing the number of times we did this would improve the speed but maybe 10times isn't much because when i just tested the speed difference with and without the return null it was almost the same so i guess I'll remove this check.